.github/CODEOWNERS

```
@@ -20,5 +20,6 @@
 /libs/@guardian/eslint-config-typescript/ @guardian/client-side-infra
 /libs/@guardian/browserlist-config/  @guardian/client-side-infra @guardian/dotcom-platform
 /libs/@guardian/libs/ @guardian/client-side-infra @guardian/apps-rendering @guardian/dotcom-platform
+/libs/@guardian/identity-auth-frontend/ @guardian/dotcom-platform @guardian/commercial-dev
```

```suggestion
/libs/@guardian/identity-auth-frontend/ @guardian/dotcom-platform @guardian/commercial-dev @guardian/identity
```

think it might be useful to have identity here too